src/aat/resources/application.yml

```diff
@@ -33,7 +33,7 @@ s2s:
   api:
     url: ${S2S_BASE_URI:http://localhost:4502}
     secret: ${S2S_KEY:AAAAAAAAAAAAAAAA}
-    serviceName: ${S2S_SERVICE_NAME:em_hrs_api}
+    serviceName: ${S2S_SERVICE_NAME:em_gw}
```

Comment:
why are we going back to using em_gw?

Reply:
Well. This is so that we don't whitelist em_hrs_api against em_hrs_api itself. That was your previous review comment